app/views/layouts/_header.html.erb

```diff
@@ -20,11 +20,19 @@
 
       <% if logged_in? %>
       <li class="nav-item">
-        <%= link_to "Users", "#", class:"nav-link"%>
+        <%= link_to "Users", users_path, class:"nav-link"%>
       </li>
       <li class="nav-item">
         <%= link_to "Logout", logout_path, class:"nav-link"%>
       </li>
+        <li><%= link_to "Profile", current_user, class:"nav-link"%></li>
+        <li><%= link_to "Settings",
```

Comment:
Please change format of your code like this if you have a start and end tag :bow: 
![image](https://user-images.githubusercontent.com/35232348/103744459-cba65b80-5038-11eb-8a01-30363d3a1572.png)